layouts/default.vue

```diff
@@ -120,7 +120,10 @@
         // Once the PWA code is initialized, this holds a method
         // that can be called to show the user the installation
         // prompt.
-        showInstallPrompt: null
+        showInstallPrompt: null,
+        logoStyle() {
+          return "margin-right: 16px;" + (this.$store.state.postwoman.settings.THEME_CLASS ? " filter: invert(100%); -webkit-filter: invert(100%);" : '')

```

Comment:
THEME_CLASS being set does not necessarily imply light theme. Can you perhaps change it to `.includes('light')`?

Reply:
Yes. Agree.
Updated the code.